src/main/java/at/ac/tuwien/kr/alpha/solver/learning/GroundConflictNoGoodLearner.java

```diff
@@ -122,17 +143,18 @@ private ConflictAnalysisResult analyzeConflictingNoGoodRepetition(NoGood violate
 			// The real conflict therefore is caused by either:
 			// a) two NoGoods propagating the same atom to different truth values in the current decisionLevel, or
 			// b) a NoGood propagating at a lower decision level to the inverse value of a choice with higher decision level.
+			// TODO: b) should no longer be possible, since all propagation is on highest decision level now.
```

Comment:
Should we `throw oops` in case b) still occurs?

Reply:
I think this could be done when addressing #139.